frontend/login.html

```diff
@@ -12,19 +12,21 @@
         <h1>Mars Rover Project</h1>
         <h2>Login</h2>
         <p>Login here to access your saved NASA images and data.</p>
-        <div class="login">
-            <div class="form-input">
-                <label for="username">Username</label>
-                <input type="text" id="username">
+        <form action="/authenticate" method="POST">
+            <div class="login">
+                <div class="form-input">
+                    <label for="username">Username</label>
+                    <input type="text" id="username" name="username">
+                </div>
+                <div class="form-input">
+                    <label for="password">Password</label>
+                    <input type="password" id="password" name="password">
+                </div>
+                <button type="submit">
+                    Submit
+                </button>
             </div>
```

Comment:
For this, do we strictly need this div?

Reply:
No I'm. not sure that we do. I think the `<form>` tags would supercede the `<div>`. I'll change it. :)

Reply:
Done :)